rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+merge_imports = true
+
+ignore = [
+    "mobilecoin",   # The mobilecoin submodule is a dependency of this project.
```

Comment:
I **think** this is the right thing to do?

Reply:
I think so.
I would suggest adding these two:
```
edition = "2018"
wrap_comments = true
```

@jcape has been using them in some places and I have created a ticket for myself to put that in the root `rustfmt.toml`